api/pkg/storage/docker_volume.go

```diff
@@ -0,0 +1,37 @@
+package storage
```

Comment:
let's not call the file `docker_volume.go` but rather `file.go`. Also, can we add an interface and some tests?

Reply:
sure! will do